README.md

```diff
@@ -53,6 +53,7 @@ There are two main categories of samples: **Camera** and **Applications**. The s
 
 [**Install Zivid Software**](https://zivid.atlassian.net/wiki/spaces/ZividKB/pages/59080712/Zivid+Software+Installation).
 Note: The version tested with Zivid cameras is 1.8.0.
+[**Download Zivid Sample Data**](https://zivid.atlassian.net/wiki/spaces/ZividKB/pages/edit-v2/450363393?draftShareId=2fd72323-1c01-4acb-8016-ed74206c4edb).
```

Comment:
Update with the link: https://zivid.atlassian.net/wiki/spaces/ZividKB/pages/450363393/Sample+Data

Reply:
done